`Header.entry_point` incorrect default value

The default value of `Header.entry_point` should be `null`
https://github.com/ominitay/zigvale/blob/1a112c14c664834058ae4fceb2669af7a7c761c9/src/stivale2.zig#L44

```
./deps/zigvale/src/stivale2.zig:44:62: error: expected type 'fn(*const .zigvale.src.stivale2.Struct) callconv(.C) noreturn', found 'comptime_int'
    entry_point: ?fn (*const Struct) callconv(.C) noreturn = 0,
                                                             ^
```

--- a/src/stivale2.zig
+++ b/src/stivale2.zig
@@ -41,7 +41,7 @@ test "TagGeneric" {
 /// The kernel must have a section `.stivale2hdr` either containing a header, or an anchor pointing to one.
 pub const Header = packed struct {
     /// The address to be jumped to as the entry point of the kernel. If 0, the ELF entry point will be used.
-    entry_point: ?fn (*const Struct) callconv(.C) noreturn = 0,
+    entry_point: ?fn (*const Struct) callconv(.C) noreturn = null,
     /// The stack address which will be in ESP/RSP when the kernel is loaded.
     /// The stack must be at least 256 bytes, and must have a 16 byte aligned address.
     stack: ?*u8,

